packages/schemas/tables/custom_ui_assets.sql

```diff
@@ -0,0 +1,12 @@
+create table custom_ui_assets (
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  files jsonb /* @use UploadFiles */ not null default '[]'::jsonb,
```

Comment:
actually i doubt if we need to store the metadata of all files - why don't we just read the directory in the storage provider..

Reply:
Table removed